components/datadog/updater/install.go

```diff
@@ -87,16 +86,20 @@ func NewHostUpdaterWithPackages(e *config.CommonEnvironment, host *remoteComp.Ho
 
 func (h *HostUpdater) installUpdater(params *agentparams.Params, packages []string, baseOpts ...pulumi.ResourceOption) error {
 	pipelineID := fmt.Sprintf("DD_PIPELINE_ID=%v", params.Version.PipelineID)
-	installCmdStr := fmt.Sprintf(`export %v && bash -c %s`, pipelineID, installScript)
-
-	for _, pkg := range packages {
-		installCmdStr = fmt.Sprintf("%s\nsudo %s bootstrap -P %s", installCmdStr, installerPath, pkg)
+	agentConfig := pulumi.Sprintf("")
+	for _, extraConfig := range params.ExtraAgentConfig {
+		agentConfig = pulumi.Sprintf("%v\n%v", agentConfig, extraConfig)
 	}
+	agentConfig = pulumi.Sprintf("AGENT_CONFIG='%v'", agentConfig)
```

Comment:
I think that doesn't work if the extra configs contain `'` .
I don't think it's worth escaping them but you should document that somewhere